src/footer/footer.jsx

```diff
@@ -30,17 +30,17 @@ const Footer = () => {
                 </div>
                 <div className='funiro_col furniro_col_second'>
                     <h3 className='links_title'>Links</h3>
-                    {<FooterLinksList elem={footerLinks[0]} />}
+                    {<FooterLinksList link={footerLinks.links} />}
```

Comment:
```suggestion
                    <FooterLinksList link={footerLinks.links} />
```
no need fur curly braces here, check other place as well